torch/__init__.py

```diff
@@ -2494,6 +2494,8 @@ def foo(x):
     _C._log_api_usage_once("torch.compile")
     if sys.version_info >= (3, 14):
         raise RuntimeError("Dynamo is not supported on Python 3.14+")
+    elif sys.version_info >= (3, 13) and not sys._is_gil_enabled():
```

Comment:
```suggestion
    elif bool(sysconfig.get_config_var('Py_GIL_DISABLED')):
```

Reply:
I think it's probably fine as written.

Reply:
If it's a build variant issue, then `sysconfig.get_config_var('Py_GIL_DISABLED')` is more appropriate. If it's just thread safety issues, then the `sys._is_gil_enabled()` is fine.